generators/blank/templates/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+node_modules
+public
```

Comment:
/public and /build

because you could have a `build` directory in your src application